packages/calendar/src/Calendar.ts

```diff
@@ -200,20 +212,26 @@ export class Calendar extends SpectrumElement {
             changedProperties.has('currentDate') &&
             (!previousDate || !isSameMonth(previousDate, this.currentDate));
 
-        if (changesMonth) this.setCurrentMonthDates();
+        if (changesMonth) {
+            this.setCurrentMonthDates();
+            this.setAttribute('aria-label', this.monthAndYear);
```

Comment:
I wonder if this is the best way to make the screen reader announce the current month-year when focusing in the Calendar. (the same method was applied for the DateTimePicker as well)